pkg/console/subresource/configmap/configmap_test.go

```diff
@@ -332,14 +382,137 @@ servingInfo:
   keyFile: /var/serving-cert/tls.key
 providers: 
   statuspageID: id-1234
+`,
+				},
+			},
+		},
+		{
+			name: "Test operator config with Statuspage pageID",
+			args: args{
+				operatorConfig: &operatorv1.Console{
+					Spec: operatorv1.ConsoleSpec{
+						OperatorSpec: operatorv1.OperatorSpec{},
+						Customization: operatorv1.ConsoleCustomization{
+							Brand:                operatorv1.BrandDedicated,
+							DocumentationBaseURL: mockOperatorDocURL,
+						},
+						Providers: operatorv1.ConsoleProviders{
+							Statuspage: &operatorv1.StatuspageProvider{
+								PageID: "id-1234",
+							},
+						},
+					},
+					Status: operatorv1.ConsoleStatus{},
+				},
+				consoleConfig: &configv1.Console{},
+				managedConfig: &corev1.ConfigMap{
+					Data: map[string]string{configKey: `kind: ConsoleConfig
+apiVersion: console.openshift.io/v1
+customization:
+  branding: online
+  documentationBaseURL: https://docs.okd.io/4.2/
+`,
+					},
+				},
+				infrastructureConfig: &configv1.Infrastructure{
+					Status: configv1.InfrastructureStatus{
+						APIServerURL: mockAPIServer,
+					},
+				},
+				rt: &routev1.Route{
+					Spec: routev1.RouteSpec{
+						Host: host,
+					},
+				},
+				downloadsRoute: &routev1.Route{
+					Spec: routev1.RouteSpec{
+						Host: downloadHost,
+					},
+				},
+			},
+			want: &corev1.ConfigMap{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:        api.OpenShiftConsoleConfigMapName,
+					Namespace:   api.OpenShiftConsoleNamespace,
+					Labels:      map[string]string{"app": api.OpenShiftConsoleName},
+					Annotations: map[string]string{},
+				},
+				Data: map[string]string{configKey: `kind: ConsoleConfig
+apiVersion: console.openshift.io/v1
+auth:
+  clientID: console
+  clientSecretFile: /var/oauth-config/clientSecret
+  oauthEndpointCAFile: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+clusterInfo:
+  consoleBaseAddress: https://` + host + `
+  masterPublicURL: ` + mockAPIServer + `
+  cliDownloadURLs:
+    linuxDownloadURL: ` + downloadLinuxURL + `
+    macDownloadURL: ` + downloadMacURL + `
+    windowsDownloadURL: ` + downloadWindowsURL + `
+customization:
+  branding: ` + string(operatorv1.BrandDedicated) + `
+  documentationBaseURL: ` + mockOperatorDocURL + `
+servingInfo:
+  bindAddress: https://0.0.0.0:8443
+  certFile: /var/serving-cert/tls.crt
+  keyFile: /var/serving-cert/tls.key
+providers: 
+  statuspageID: id-1234
+`,
+				},
+			},
+		},
+		{
+			name: "Test openshift-console namespace without downloads route",
```

Comment:
+1 for a test `without` the downloads route.
`Test openshift-console namespace without downloads route` - `namespace`?  I'm a bit confused by the test name, however.

Reply:
Wasnt really sure how to name the test, other then this since the setup is `nil` download route. But really should be something like : `Test getting URLs for downloading CLI, without downloads route.` ... maybe ?

Reply:
The expectation is that there would be no downloads then, if there is no route, correct?  I'm good with long tests names that clearly indicate everything, including expected output.

Perhaps `Validate that there is no download CLI links when download route is `nil``?

Out test naming convention isn't super strong, so if we have to break it for clarity, thats fine.  We could always go back and improve the names of the other tests if you have something that works better.